### PR TITLE
Add auto linter for json files

### PR DESCRIPTION
**Pull request by PysarenkoDS-BWT**
Add eslint-plugin-json to lint json files

### CODE REVIEW

**Comment by ericboucher** [comment]
When/how do we run it? I'm surprised it doesn't change anything in the existing JSON files

**Comment by PysarenkoDS-BWT** [comment]
Add script to run prettier for JSON files, run only manually (not add automatically), because this not fast operation

**Comment by ericboucher** [comment]
@PysarenkoDS-BWT thanks for updating the files. Please make sure that the CI tests are failing when the files are not properly formated

**Comment by PysarenkoDS-BWT** [comment]
@ericboucher I think we can't check it in CI/CD because for this check need a loot of memory, remove this check?

**Review by ericboucher — CHANGES_REQUESTED** [review]
@PysarenkoDS-BWT actually, let's not run this on files that are in public/data/COUNTRY_NAME but only on config files for now. This is making all the geojson files even bigger, which is clearly not what we want, thanks :)

File: src/components/MapView/AlertForm/index.tsx [hunk 1]
```diff
@@ -111,35 +112,35 @@ function AlertForm({ classes, isOpen, setOpen }: AlertFormProps) {
...
   };
 
-  const onOptionChange = <T extends string>(
-    setterFunc: Dispatch<SetStateAction<T>>,
-  ) => (event: React.ChangeEvent<HTMLInputElement>) => {
-    const value = event.target.value as T;
-    setterFunc(value);
-    return value;
-  };
+  const onOptionChange =
```
Comment (ericboucher) [inline src/components/MapView/AlertForm/index.tsx:115]: Why are the ts and tsx files changing? Can we limit the changes to  json files?

**Review by ericboucher — APPROVED** [review]

**Pull request merged**